### ISSUE TITLE
@cssprop default value is missing from viewer

### ISSUE THREAD

**Issue by juliexiong** [web-padawan/api-viewer-element#26]
When I provide a default value for a CSS custom property, it does not appear in the API viewer.

When I document this:

```js
/**
* @cssprop [--card-background-color=white] - Background color for card
**/
```

It only outputs the CSS property and description:
![image](https://user-images.githubusercontent.com/3359330/75287651-5c3cca00-57e0-11ea-9e4f-fd4874fe3775.png)

**Comment by web-padawan** [comment]
Thanks for the issue. This syntax was added to `web-component-analyzer` after the `@cssprop` support was implemented here in `api-viewer`. I'm going to add default values support.

Added this to the milestone for the upcoming 0.4.0 release (hope to deliver within a week).

**Comment by web-padawan** [comment]
Done by #30 and will be included to the upcoming `0.4.0` release.
